### DISCUSSION TITLE
Sunbird ED 4.10.0.0 Release Plan

### DISCUSSION THREAD

**Discussion by chitranshu-keshav** [Sunbird-Ed/Community#8] [General Discussion]
The next stable release of Sunbird ED will be 4.10.0.0 and the important dates for the same can be found [**here**](https://ed.sunbird.org/use/releases/release-calendar-4.9-4.10-5.0).

Sunbird ED is expecting the next release version of all the other building blocks to be available by 6th of June for it to integrate in its 4.10.0.0 release.  Below are the important milestones and its dates that other building block are working towards for their 4.10.0.0 release

![image](https://user-images.githubusercontent.com/47350594/169027830-9f6070af-983b-4ae0-a58b-a960d4838d03.png)

For 4.10.0.0 release please ensure below fields are updated for the issues in JIRA 
Sprint = "4.10.0.0 Sprint"
Building Block = The respective Building Block
Contributor = The respective contributor
Field "Development Ready" = Yes"
Field "Engineering Ready" = "Yes" OR "Picked up in Sprint" 

Below are contact for individual BB's. You can refer them for any questions specific to a particular building block
Sunbird ED @luckynvdu @sharathkashyap
Sunbird Knowlg  @Krishnaj20 @vinukumar-vs
Sunbird Inquiry @Krishnaj20 @pallakartheekreddy
Sunbird CoKreate @Krishnaj20 @pallakartheekreddy
Sunbird Lern @rhwarrier @reshmi-nair
Sunbird Observ @rhwarrier @anandp504

Shikshalokam: @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30  @varsha-sl @MeghnaSL
Samagra: @Prerna789 @aayushkedia95
SunbirdCB: @pkranga
Avanti: @pritamps @karn-nahata
DC's: @SanthoshVasabhaktula @vrayulu
Sunbird QA: @ashokreddy1208
@gandham-santhosh
@kameshbhr @mmanimegalai
@madhucr @alok-os

**Comment by chitranshu-keshav** [comment]
There is a new environment created for the building blocks other than Sunbird ED & Cokreate. This environment can be accessed via https://dev.sunbird.org/ . Any PR raised against these building blocks repositories (Lern, Knowlg, Inquiry & Obsrv) will get build and deployed to this new environments. If you would need access to Jenkins or infra please raise a request in [#devops](https://ekstep.slack.com/archives/C2R6ABXRA) slack channel.

Please Note that there is no change for Sunbird ED and Sunbird Cokreate Building Blocks

Please reach out to the TLs of individual building block any additional details. 
**Building Block TLs**
Sunbird Knowlg @vinukumar-vs
Sunbird Inquiry @pallakartheekreddy
Sunbird Lern @reshmi-nair
Sunbird Observ @anandp504
Sunbird ED @sharathkashyap
Sunbird CoKreate @pallakartheekreddy

Shikshalokam: @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL
Samagra: @Prerna789 @aayushkedia95
SunbirdCB: @pkranga
Avanti: @pritamps @karn-nahata
DC's: @SanthoshVasabhaktula @vrayulu
PL: @luckynvdu @Krishnaj20 @rhwarrier
Sunbird QA: @ashokreddy1208
@gandham-santhosh
@kameshbhr @mmanimegalai
@madhucr @alok-os

**Comment by sharathkashyap** [comment]
Kindly help us with release notes of respective building blocks

Sunbird ED @luckynvdu @sharathkashyap
Sunbird Knowlg @Krishnaj20 @vinukumar-vs
Sunbird Inquiry @Krishnaj20 @pallakartheekreddy
Sunbird CoKreate @Krishnaj20 @pallakartheekreddy
Sunbird Lern @rhwarrier @reshmi-nair
Sunbird Observ @rhwarrier @anandp504


DC's: @SanthoshVasabhaktula @vrayulu
Sunbird QA: @ashokreddy1208
@gandham-santhosh
@kameshbhr @mmanimegalai
@madhucr @alok-os

cc: @chitranshu-keshav @luckynvdu

**Reply by pallakartheekreddy** [reply]
@sharathkashyap Please find below Sunbird inQuiry release notes for 4.10.0 
https://inquiry.sunbird.org/use/release-notes/inquiry-release-v-4.8.0-upcoming

**Reply by reshmi-nair** [reply]
@sharathkashyap Please find the release notes for Sunbird Lern release-4.10.0 
https://lern.sunbird.org/use/release-notes/release-v-4.10.0-latest

**Comment by vinukumar-vs** [comment]
@sharathkashyap @luckynvdu SB-Knowlg latest release(4.10) notes
https://knowlg.sunbird.org/use/release-notes/latest

**Reply by rhwarrier** [reply]
Release notes for SB Obsrv:
https://obsrv.sunbird.org/use/release-notes 
cc: @anandp504
